src/features/user/Account/History.tsx

```diff
@@ -60,6 +78,46 @@ export default function History({
     }
   }, [paymentHistory]);
 
+  const handleIssueReceipts = async () => {
+    setLoading(true);
+    try {
+      const res = await postAuthenticatedRequest(
+        '/app/taxReceipts',
+        {
+          year: new Date().getFullYear(),
+        },
+        token,
+        handleError
+      );
+      setLoading(false);
+      if (res && res.length === 0) {
+        setError({
+          type: 'error',
+          message: t('me:taxReceiptsAlreadyGenerated'),
+        });
+        setSelectedRecord(0);
```

Comment:
Not clear why `setSelectedRecord(0)` is being called after receiving an error. Perhaps the page should be left in the same state as it was?

Reply:
Fixed here https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1677/commits/afc97e7af345ab47562cea884bb41b04efdb5e70